api/HTMLMarqueeElement.json

```diff
@@ -438,10 +438,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "65"
+              "version_added": false
```

Comment:
Firefox and Chrome have `trueSpeed` instead, so maybe just a typo.

Reply:
Argh, we seem to have a lot of these miscapitalized entries...

Reply:
Yep :) You've already opened https://github.com/foolip/mdn-bcd-collector/issues/812, and doing that soon would be good, as now we're relying on the reviewer to spot suspicious changes.